docs/superpowers/specs/2026-04-09-stackflow-overlay-display-design.md

```
@@ -0,0 +1,172 @@
+  activityType: ActivityType;
+  transitionState: string;
+}
+
+// connections: Map<zIndex, HideEffectConnection>
+```
+
+### Hide Logic Algorithm
+
+스택을 top(가장 높은 zIndex)에서 bottom으로 순회하며 display를 결정한다:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)